air/tests/test_module/negative_tests/uncatchable_trace_related.rs

```diff
@@ -321,7 +323,7 @@ fn invalid_dst_generations() {
     let mut data_value = serde_json::to_value(&empty_data).unwrap();
     data_value["trace"] = json!([{"ap": {"gens": [42, 42]}}]);
 
-    let data: Vec<u8> = serde_json::to_vec(&data_value).unwrap();
+    let data = InterpreterDataRepr.get_format().to_vec(&data_value).unwrap();
```

Comment:
I think it better to name it `decoder` instead of a `format`, it'll improve code readability.

Reply:
This entity both encodes and decodes, "decoder" is not a suitable name.
As we discussed privately, this is low-level API designed to be able to deal with arbitrary values (including invalid ones) in tests.